css/properties/-webkit-max-logical-width.json

```diff
@@ -5,7 +5,7 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": "≤80"
+              "version_added": "≤15"
```

Comment:
❓ This property is not reported by computed properties, so I'm not sure about this one.